maven/bnd-maven-plugin-lib/.project

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>bnd-maven-plugin-lib</name>
```

Comment:
Lets change the name to `bnd-shared-maven-lib` unless we put all this code in parent!